### DISCUSSION TITLE
att&ck navigator 'view related plays' does always open T1001 instead of actual playbook so version 2.3.80

### DISCUSSION THREAD

**Discussion by fcblank** [Security-Onion-Solutions/securityonion#5766] [General]
when right klicking 'view related play' in the att&ck navigator with an active play behind, the link to the playbook doesn't work. The called url to the detection playbook looks like this:
https://10.1.1.1/playbook/projects/detection-playbooks/issues?utf8=✓&set_filter=1&sort=id:desc&f[]=cf_15&op[cf_15]==&f[]=&c[]=status&c[]=cf_10&c[]=cf_13&c[]=cf_18&c[]=cf_19&c[]=cf_1&c[]=updated_on&v[cf_15][]=**~Technique_ID~**
Please note, that the technique_id is not passed.
Is this a bug?

**Comment by defensivedepth** [comment]
This may be a bug, will need to confirm.
